nautobot/ipam/views.py

```diff
@@ -206,7 +206,10 @@ class RIRBulkDeleteView(generic.BulkDeleteView):
 class AggregateListView(generic.ObjectListView):
     queryset = Aggregate.objects.annotate(
         child_count=RawSQL(
-            "SELECT COUNT(*) FROM ipam_prefix WHERE ipam_prefix.prefix <<= ipam_aggregate.prefix",
+            "SELECT COUNT(*) FROM ipam_prefix "
+            "WHERE ipam_prefix.prefix_length >= ipam_aggregate.prefix_length "
+            "AND ipam_prefix.network >= ipam_aggregate.network "
+            "AND ipam_prefix.broadcast <= ipam_aggregate.broadcast",
```

Comment:
While we're in the guts, is it reasonable to switch this to an ORM queryset and ditch the raw SQL?

Reply:
The ORM doesn't support this query

Reply:
Oh right because of the weird group by issue in subqueries?

Reply:
Yes